data-raw/qc-markdowns/temperature_data_prep.Rmd

```diff
@@ -718,8 +721,8 @@ write_rds(MLM_daily_temps, here::here("data-raw", "standard-format-data-prep","t
 ### CDEC
```

Comment:
Remaining TODO for Yuba, we need to add a Recommendations section

Reply:
We should pull the USGS 114118000 data too (unless I am forgetting reason not to).